openstack/intel-worker/tasks/ignition.yml

```
@@ -28,3 +28,24 @@
     template:
       src: ../templates/worker-amd64.ign.j2
       dest: /tmp/worker-amd64.ign
+
+  - name: "Download the Image from the URL and use to create image"
+    ansible.builtin.get_url:
+      url: "{{ image_url }}"
+      dest: "{{ image_dest_filename  }}"
+      mode: '0755'
+      validate_certs: false
```

```suggestion
      validate_certs: false
```
Let's have the validate_certs come be a var

---

Sure added to vars , thanks